containers/R/rclipper_user/R/build.R

```diff
@@ -0,0 +1,106 @@
+#' Builds a Docker image for a Clipper model container that can be launched to serve a model
+#' with the provided prediction function
+#' 
+#' @param model_name string (character vector of length 1). The name to assign to the model image.
+#' @param model_version string (character vector of length 1). The version tag to assign to the model image.
+#' @param prediction_function function. This should accept a type-homogeneous list of 
+#' inputs and return a list of outputs of the same length. If the elements of the output list
+#' are not strings (character vectors of length 1), they will be converted to a serialized
```

Comment:
R doesn't have a string datatype, so I was referring to length-1 character vectors as strings. I've removed any references to "string" because it's confusing. Unfortunately, there's no type distinction between a single character array and a vector of character arrays in R's type system, so the "length-1" specification is necessary.